fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -2471,13 +2466,6 @@ public void modifyTableLightSchemaChange(Database db, OlapTable olapTable,
             throws DdlException {
 
         LOG.debug("indexSchemaMap:{}, indexes:{}", indexSchemaMap, indexes);
-        if (olapTable.getState() == OlapTableState.ROLLUP) {
-            throw new DdlException("Table[" + olapTable.getName() + "]'s is doing ROLLUP job");
-        }
-
-        // for now table's state can only be NORMAL
-        Preconditions.checkState(olapTable.getState() == OlapTableState.NORMAL, olapTable.getState().name());
-
```

Comment:
Is ok removing this check here? I think here also need `olapTable.checkNormalStateForAlter();`

Reply:
we already check table state inside the table write lock no need double check here